samples/Azure.AI.DocumentTranslation/Generated/DocumentTranslationClient.cs

```diff
@@ -35,16 +35,22 @@ protected DocumentTranslationClient()
         {
         }
 
+        /// <summary> Initializes a new instance of DocumentTranslationClient. </summary>
+        /// <param name="endpoint"> Supported Cognitive Services endpoints (protocol and hostname, for example: https://westus.api.cognitive.microsoft.com). </param>
+        /// <param name="credential"> A credential used to authenticate to an Azure Service. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="endpoint"/> or <paramref name="credential"/> is null. </exception>
+        public DocumentTranslationClient(string endpoint, AzureKeyCredential credential) : this(endpoint, credential, new DocumentTranslationClientOptions())
+        {
+        }
         /// <summary> Initializes a new instance of DocumentTranslationClient. </summary>
         /// <param name="endpoint"> Supported Cognitive Services endpoints (protocol and hostname, for example: https://westus.api.cognitive.microsoft.com). </param>
         /// <param name="credential"> A credential used to authenticate to an Azure Service. </param>
         /// <param name="options"> The options for configuring the client. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="endpoint"/> or <paramref name="credential"/> is null. </exception>
-        public DocumentTranslationClient(string endpoint, AzureKeyCredential credential, DocumentTranslationClientOptions options = null)
+        public DocumentTranslationClient(string endpoint, AzureKeyCredential credential, DocumentTranslationClientOptions options)
         {
             Argument.AssertNotNull(endpoint, nameof(endpoint));
             Argument.AssertNotNull(credential, nameof(credential));
-            options ??= new DocumentTranslationClientOptions();
```

Comment:
Why is this line removed?  It seems possible that someone could pass null to the constructor and we would want to create an options instance.

Reply:
This line is removed because `options` is required parameter now. If they want to pass null, they should call `public DocumentTranslationClient(string endpoint, AzureKeyCredential credential)`, right?

Reply:
This is incorrect functionality.  The client won't function correctly if `options` is null.  If a customer passes null for options, we should instantiate a default instance of the client options for the library.